src/plugins/saved_objects_management/public/management_section/objects_table/components/import_summary.tsx

```diff
@@ -176,13 +184,85 @@ const getStatusIndicator = ({ outcome, errorMessage = 'Error' }: ImportItem) =>
   }
 };
 
-export const ImportSummary = ({ failedImports, successfulImports }: ImportSummaryProps) => {
-  const importItems: ImportItem[] = _.sortBy(
-    [
-      ...failedImports.map((x) => mapFailedImport(x)),
-      ...successfulImports.map((x) => mapImportSuccess(x)),
-    ],
-    ['type', 'title']
+const ImportWarnings: FC<{ warnings: SavedObjectsImportWarning[]; basePath: IBasePath }> = ({
+  warnings,
+  basePath,
+}) => {
+  if (!warnings.length) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiSpacer size="m" />
+      {warnings.map((warning, index) => (
+        <Fragment key={`warning-${index}`}>
+          <ImportWarning warning={warning} basePath={basePath} />
+          {index < warnings.length - 1 && <EuiSpacer size="s" />}
+        </Fragment>
+      ))}
+    </>
+  );
+};
+
+const ImportWarning: FC<{ warning: SavedObjectsImportWarning; basePath: IBasePath }> = ({
+  warning,
+  basePath,
+}) => {
+  const warningContent = useMemo(() => {
+    if (warning.type === 'action_required') {
+      return (
+        <EuiFlexGroup alignItems="flexEnd" justifyContent="flexEnd" gutterSize="s">
+          <EuiFlexItem grow={false}>
+            <EuiButton
+              size="s"
+              color="warning"
+              href={basePath.prepend(warning.actionUrl)}
+              target="_blank"
+            >
+              {warning.buttonLabel || (
+                <FormattedMessage
+                  id="savedObjectsManagement.importSummary.warnings.defaultButtonLabel"
+                  defaultMessage="Go"
+                />
+              )}
+            </EuiButton>
```

Comment:
The warning are displayed in the import flyout, meaning that navigating to the target of an action warning and going back to SO management would result on the flyout to no longer be present. This is an issue if there are multiple warnings requiring an action. This is why I used a `target="_blank"` on the link here, to always open the warning's target page in another tab. This is not ideal in an SPA, but that was the only solution I found that did not required fundamental changes on the SO import flyout / workflow. I guess this is alright for now.